app/Keycloak/Converters/IntegrationUrlConverter.php

```diff
@@ -4,62 +4,56 @@
 
 namespace App\Keycloak\Converters;
 
+use App\Domain\Integrations\Environment;
 use App\Domain\Integrations\Integration;
 use App\Domain\Integrations\IntegrationPartnerStatus;
 use App\Domain\Integrations\IntegrationUrl;
 use App\Domain\Integrations\IntegrationUrlType;
-use App\Keycloak\Client;
 
 /* Converts integration urls in the correct Keycloak API format */
+
 final readonly class IntegrationUrlConverter
```

Comment:
To avoid future use it is maybe best to move these methods as private methods on `IntegrationToKeycloakClientConverter`. This will prevent developers from using `IntegrationUrlConverter` directly.

Reply:
I would prefer not to do this.
1) The argument that a developer could use the class directly in error can be made for any splitting up of code in different classes. That is the downside of the Single Responsibility Principle, and the downside of a language where you always export everything and cannot hide or seal some classes in a namespace.
2) When I started doing this I quickly realised that this would make testing several cases in IntegrationUrlConverterTest a lot more difficult. Because you can no longer target the url creation directly the short, clear defined unit tests would become really big.